sys/net/application_layer/asymcute/asymcute.c

```diff
@@ -517,15 +517,20 @@ static void _on_unsuback(asymcute_con_t *con, const uint8_t *data, size_t len)
 static void _on_data(asymcute_con_t *con, size_t pkt_len, sock_udp_ep_t *remote)
 {
     size_t len;
-    size_t pos = _len_get(con->rxbuf, &len);
+    size_t pos;
```

Comment:
why not add the check ( `if (pkt_len < 2) ...` ) right before the variable declarations? RIOT is using C99 and above so no need to declare all vars up front.